CONTRIBUTING.md

```diff
@@ -1,54 +1,87 @@
+```
 
+For more info, see [here](http://wiki.eclipse.org/Development_Resources/Contributing_via_Git). Once all issues brought by committers are addressed it can be landed by any committer.
 
+In the case of an objection being raised in a pull request by another committer, all involved  committers should seek to arrive at a consensus by way of addressing concerns being expressed  by discussion, compromise on the proposed change, or withdrawal of the proposed change.
+
+If a contribution is controversial and committers cannot agree about how to get it to land or if it should land then it should be escalated to the [commiters group](https://groups.google.com/forum/#!forum/vertx-committers). Members of the developers group should regularly discuss pending contributions in order to find a resolution. It is expected that only a small minority of issues be brought to the group for resolution and that discussion and compromise among committers be the default resolution mechanism.
+
+
+# Becomming a Contributor
+
+Becoming a contributor to the project should be a easy step. In order to reduce that barrier new contributors should look for issues tagged as:
+
+* `Quick Win`
+* `Help Wanted`
+
+These tags should be applied to issues that are relatively simple to fix but not critical to be handled by the current group of committers. Once you pick such an issue to work on a discussion with a committer should happen on the issue itself and the committer should guide you with the mentoring/onboarding process.
+
+
+# Becoming a Committer
+
+All contributors who land a non-trivial contribution should be on-boarded in a timely manner, and added as a committer, and be given write access to the repository. 
+
+Committers are expected to follow this policy and continue to send pull requests, go through proper review, and have other committers merge their pull requests.
+
+
+# Mentoring / Onboarding
+
+Commiters should help mentor/on-board new contributors. The workflow should roughly follow this items:
+
+* suggest the new contributor to join the [development group](https://groups.google.com/forum/#!forum/vertx-dev)
+* ask for participation on github issue discussions
+* suggest to keep in touch with on the dev group on periodic way (e.g.: weekly)
+* help finding a good mentor if one is not well versed in a specific are of the project
+* suggest progress reporting
+* assist with creating an Eclipse user id and ECA sign
+
+
+# Technical Conflict Process
+
+The Vert.x project uses a "consensus seeking" process for issues that are escalated to the [commiters group](https://github.com/orgs/vert-x3/people). The group tries to find a resolution that has no open objections among the members. If a consensus cannot be reached that has no objections then a majority wins vote is called. It is also expected that the majority of decisions made by the group are via a consensus seeking process and that voting is only used as a last-resort.
```

Comment:
i did not do that because if i remember correctly that goes against the eclipse rules. I think we had discussed it the past... I'm fine in adding that